docs/kratos/hooks/01_configure-hooks.mdx

```diff
@@ -0,0 +1,417 @@
+---
+id: configure-hooks
+title: Add and configure hooks for self-service user flows
+sidebar_label: Configuration
+toc_max_heading_level: 4
+---
+
+Ory Identity Service (Kratos) allows you to enrich self-service user flows by executing additional logic at specific points of the
+selected flow. You enrich the flows through hooks.
+
+Adding hooks to self-service flows can help you shape your customers' user experience and increase the security of your systems.
```

Comment:
Would love to mention the use case of 3P integration here, and provide a reference to the webhook guidance. WDYT?

"Adding hooks to self-service flows can help you shape your customers' user experience, increase the security of 
your systems through additional verifications, and exchange information with external systems"

Reply:
> Would love to mention the use case of 3P integration here
What do you mean by "3P"? I don't think I understand. 

If you want the sentence to sound the way you suggest then sure, we can go in that direction.

Reply:
Does 3P = 3rd party?

Reply:
yes